index.js

```diff
@@ -0,0 +1,274 @@
+/**
+ * Full fhirconvert function: validates, converts, then identifies variables.
+ * @returns converted fhirpath expression
+ */
+export function fhirconvert(str, vars, funs) {
+    if (validate(str, vars, funs)){
+        return varfind(convert(str), vars);
+    }
+    else {
+        return null;
+    }
+}
+
+/**
+ * Verifies normal syntax by confirming var names, function names,
+* and number of parenthesis.
+* @param {string} str - inputted normal syntax expression
+* @returns boolean, valid or invalid
+*/
+export function validate(str, vars, funs) {
+var lcount = 0;
```

Comment:
Indentation issue:  Inside a block, there should be two spaces on indent per level.